analysis/analysis.go

```diff
@@ -260,6 +267,21 @@ func parseMetricInterval(logCtx log.Entry, metricDurationString v1alpha1.Duratio
 	return metricInterval, nil
 }
 
+// resolveArgsFromValueFromFieldRef resolves args when using ValueFrom.FieldRef.FieldPath
+// returns argument with default value from lookup
+func resolveArgsFromValueFromFieldRef(arg v1alpha1.Argument, namespace string) v1alpha1.Argument {
+	if arg.ValueFrom.FieldRef != nil && arg.ValueFrom.FieldRef.FieldPath != "" {
+		switch arg.ValueFrom.FieldRef.FieldPath {
+		case "metadata.namespace":
```

Comment:
Does this mean that we will only support metadata.namespace for fieldPath lookups?